### ISSUE TITLE
Kill bootlogd in runlevel 2 if service is not enabled

### ISSUE THREAD

**Issue by Tank-Missile** [cloux/runit-base#1]
While this can be changed directly in the runlevel files themselves, by adding in options to satisfy different use cases, it makes it much easier to update your modifications.

**Comment by cloux** [comment]
you really don't like the bootlogd running, don't you :-P

**Comment by Tank-Missile** [comment]
Sorry, I guess I have this stigma where I want to be able to update a developer's changes without overwriting my own customizations. I do agree there are more important issues to fix, but I just wanted to slap this on here for future reference.

**Comment by cloux** [comment]
Solved by commit 031db73465b62ba3e62797de80d51df4dfe8ca7c

To exit bootlogd after boot, run:
```
svdeactivate bootlogd
chmod 755 /etc/runit/autorun/bootlogd
```
NOTE: if the bootlogd service is marked as down (file /etc/sv/bootlogd/down exists), bootlogd will not be started at boot as well.
